tests/spec/fs.truncate.spec.js

```diff
@@ -192,3 +192,22 @@ describe('fs.truncate', function() {
     });
   });
 });
+
+
+describe('fsPromises.truncate', function () {
+  beforeEach(util.setup);
+  afterEach(util.cleanup);
+  it('should error when path does not exist (with promises)', (done) => {
+    var fsPromises = util.fs().promises;
+
+    fsPromises.truncate('/NonExistingPath', 0)
+      .then(result => expect(result).not.to.exist)
```

Comment:
I think you can delete this line.

Reply:
Should this be a `.catch(...)` error check instead?